django/db/backends/mysql/operations.py

```diff
@@ -62,7 +62,7 @@ def date_extract_sql(self, lookup_type, sql, params):
             # EXTRACT returns 1-53 based on ISO-8601 for the week number.
             lookup_type = lookup_type.upper()
             if not self._extract_format_re.fullmatch(lookup_type):
-                raise ValueError(f"Invalid loookup type: {lookup_type!r}")
```

Comment:
Can we add a test for this message? (in the `tests/backends/mysql/test_operations.py`)

Reply:
Srue